flashinfer/rope.py

```diff
@@ -1303,3 +1402,218 @@ def rope_quantize_fp8(
     )
 
     return q_rope_out, k_rope_out, q_nope_out, k_nope_out
+
+
+def rope_quantize_fp8_append_paged_kv_cache(
+    q_rope: torch.Tensor,
+    k_rope: torch.Tensor,
+    q_nope: torch.Tensor,
+    k_nope: torch.Tensor,
```

Comment:
Seems like vLLM uses these tensors directly as query, key rather than slicing the tensors. Is this strictly necessary from the kernel?